flink/v1.17/flink/src/test/java/org/apache/iceberg/flink/sink/TestIcebergStreamWriter.java

```diff
@@ -376,7 +376,7 @@ private OneInputStreamOperatorTestHarness<RowData, WriteResult> createIcebergStr
             icebergTable, Maps.newHashMap(), new org.apache.flink.configuration.Configuration());
 
     IcebergStreamWriter<RowData> streamWriter =
-        FlinkSink.createStreamWriter(icebergTable, flinkWriteConfig, flinkRowType, null);
+        FlinkSink.createStreamWriter(icebergTable, null, flinkWriteConfig, flinkRowType, null, 0);
```

Comment:
I was thinking whether it would make sense to always default to `0` rather than having to pass it explicitly

Reply:
That would only be used by the tests, as it is always specified otherwise. Do you still feel we should add that?